test/index.js

```diff
@@ -76,100 +76,122 @@ var Integer = $.NullaryType(
 );
 
 
-var MaybeTypeDict = {
-  'fantasy-land/empty': function() { return Nothing; },
-  'fantasy-land/of': function(x) { return Just(x); }
+function _Maybe(tag, value) {
```

Comment:
Would be nice if we move this types in FL as we have them in sanctuary, sanctuary-dev and i think in sanctuary-type-classes too